app/src/main/res/layout/widget_media_controls.xml

```diff
@@ -57,58 +57,83 @@
                 android:layout_alignParentTop="true"
                 android:layout_alignParentEnd="true"
                 android:padding="8dp"
-                android:visibility="invisible"
```

Comment:
Whoops, didn't know about this feature, i guess this is just one more think that samsung removed because my phone doesn't do it -and my emulator was running on 11.

Reply:
![image](https://user-images.githubusercontent.com/38779449/169405085-c0893d6e-9209-4fca-9a0b-32798f8b6e33.png)
fixed :)